src/main/java/core/db/DBUpdater.java

```diff
@@ -123,6 +123,10 @@ private void updateDBv400(int dbVersion, int version) throws SQLException {
 		m_clJDBCAdapter.executeUpdate("DELETE FROM USERCONFIGURATION WHERE CONFIG_KEY = 'aufstellungsPanel_horizontalRightSplitPane'");
 		m_clJDBCAdapter.executeUpdate("DELETE FROM USERCONFIGURATION WHERE CONFIG_KEY = 'aufstellungsPanel_horizontalLeftSplitPane'");
 
+		if ( !columnExistsInTable("SeasonOffset", BasicsTable.TABLENAME)){
+			m_clJDBCAdapter.executeUpdate("ALTER TABLE BASICS ADD COLUMN SeasonOffset INTEGER");
```

Comment:
here should we put a default value to 0 to be on the safe side ?

Reply:
@akasolace why do you think a default 0 would be a good idea?

Reply:
@wsbrenk we had quite a lot of problems with the database, when entering new column and having some null values. 
I see you treat the case with a catch/try block but maybe it is safer to avoid the null value all together.
Otherwise, you can put a dummy value like -1 and if you have -1 when getting the value, the code try to redownload worlddetails .... but I leave it up to you ... your call

Reply:
-1 would be quiet better than 0, since 0 is a valid number, which gives in countries other than sweden, france or germany and some others wrong information. but why isn't null not the perfect solution for that?
the try catch block you mentioned is a copy paste relict, i'm not very happy with.

Reply:
@wsbrenk ok fine, let's keep it as is ..... I go ahead with the merge then ....